Data Prep Code.Rmd

```diff
@@ -0,0 +1,317 @@
+---
+title: "Data Prep code"
+author: "Yamini Kuntal"
+date: "2024-05-30"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+```{r Loading Packages & Data}
+# For data manipulation
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(tidyr)
+library(knitr)
+
+df <- read_csv("E:/University Content/RR/data_DMDDG.csv")

```

Comment:
Could you please take a look at [this](https://github.com/DominikZabinski/rr_group_project/blob/main/supplementary.Rmd#L96-L102) part of code and change your so we can avoid reading files from local system and having hard-coded file paths? :)

Reply:
Done